### PR TITLE
Fix text overlap in conversation

### PR DESCRIPTION
**Pull request by chrisnojima**
add scrollbar size to measurer so the wrapping is measured correctly

### CODE REVIEW

File: shared/chat/conversation/list.desktop.js [hunk 1]
```diff
@@ -213,7 +215,7 @@ class ConversationList extends Component<void, Props, State> {
     let y = clientRect.top - (message.followState === 'You' ? 200 : 116)
     if (y < 10) y = 10
 
-    const popupComponent = this._renderPopup(message, {position: 'absolute', top: y, left: x})
+    const popupComponent = this._renderPopup(message, {left: x, position: 'absolute', top: y})
```
Comment (chrisnojima) [inline shared/chat/conversation/list.desktop.js]: 🔀 sort props

File: shared/chat/conversation/list.desktop.js [hunk 2]
```diff
@@ -259,7 +261,7 @@ class ConversationList extends Component<void, Props, State> {
   render () {
     if (!this.props.validated) {
       return (
-        <div style={{...globalStyles.flexBoxColumn, flex: 1, alignItems: 'center', justifyContent: 'center'}}>
+        <div style={{...globalStyles.flexBoxColumn, alignItems: 'center', flex: 1, justifyContent: 'center'}}>
```
Comment (chrisnojima) [inline shared/chat/conversation/list.desktop.js]: 🔀 sort props

File: shared/chat/conversation/list.desktop.js [hunk 3]
```diff
@@ -331,7 +334,7 @@ class ConversationList extends Component<void, Props, State> {
             </CellMeasurer>
           }}
         </AutoSizer>
-        {this.props.sidePanelOpen && <div style={{...globalStyles.flexBoxColumn, position: 'absolute', right: 0, top: 0, bottom: 0, width: 320}}>
+        {this.props.sidePanelOpen && <div style={{...globalStyles.flexBoxColumn, bottom: 0, position: 'absolute', right: 0, top: 0, width: 320}}>
```
Comment (chrisnojima) [inline shared/chat/conversation/list.desktop.js]: 🔀 sort props

**Comment by chrisnojima** [comment]
@keybase/react-hackers

File: shared/chat/conversation/list.desktop.js [hunk 4]
```diff
@@ -27,6 +27,8 @@ type State = {
   selectedMessageID?: MessageID,
 }
 
+const scrollbarWidth = 20
```
Comment (chromakode) [inline shared/chat/conversation/list.desktop.js]: Should we in the future detect the scrollbar width? Will this be accurate for macOS users who have enabled overlay scrollbars?

**Comment by chrisnojima** [comment]
There isn't a perfect way to do this so I think this is a safe assumption.

**Pull request merged**